config/jobs/kubernetes/sig-auth/sig-auth-encryption-at-rest.yaml

```diff
@@ -1,18 +1,18 @@
 presubmits:
   kubernetes/kubernetes:
   - name: pull-kubernetes-e2e-kind-kms
+    cluster: k8s-infra-prow-build
     decorate: true
     decoration_config:
       timeout: 150m
     always_run: false
-    optional: true
+    optional: false
```

Comment:
Hmm, we should not be jumping to adding yet another required presubmit.

Is this in release blocking yet? Is it stable? Are there frequent bad source code changes caught by this such that release blocking is insufficient coverage?

presubmits on every k/k push are not cheap to run and introduce more noise for contributors.

Reply:
@BenTheElder is the `run_if_changed` not sufficient to prevent noise?

Reply:
> Is this in release blocking yet? Is it stable? Are there frequent bad source code changes caught by this such that release blocking is insufficient coverage?

The tests are stable. We have added it to release blocking as part of this PR. Also, this only runs if the KMS files are changed, so it wouldn't block every PR?

Also, the change to add it as release blocking was done after the unit tests failed while making `optional: false`. Essentially we want this job to be required and passing when the files in `run_if_changed` are modified.

Reply:
maybe?

but we have a relatively blanket push back against new presubmits that have not yet met the criteria here: https://github.com/kubernetes/sig-release/blob/master/release-blocking-jobs.md

(which testing co-authored with SIG release)